src/main/java/androidx/core/view/View.kt

```diff
@@ -286,3 +286,18 @@ inline fun <reified T : ViewGroup.LayoutParams> View.updateLayoutParams(block: T
     block(params)
     layoutParams = params
 }
+
+/**
+ * Setting both x and y scale to view
+ *
+ * ```
+ * view.scale = 1.2f
+ * ```
+ * Returns the minimum scale of scaleY and scaleX
+ */
+var View.scale: Float
```

Comment:
I'm not convinced this is a common thing to do on View. Animations are the main use case for scale and this doesn't help animations.

Reply:
@romainguy ok, thanks. Then I thing ViewPropertyAnimator.scale is a good idea.

Reply:
Please file a feature request for that at http://b.android.com/new